pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -21,6 +21,8 @@ const (
 
 	RouterUsername   = "system:" + RouterUnqualifiedUsername
 	RegistryUsername = "system:" + RegistryUnqualifiedUsername
+
+	BuildControllerUsername = "system:serviceaccount:openshift-infra:build-controller"
```

Comment:
the name of the build controller might be configurable in the future as well... my example commit insulates you from that